server/src/main/java/org/tctalent/server/service/db/impl/CandidateServiceImpl.java

```diff
@@ -78,54 +78,12 @@
 import org.tctalent.server.exception.SalesforceException;
 import org.tctalent.server.exception.UsernameTakenException;
 import org.tctalent.server.logging.LogBuilder;
-import org.tctalent.server.model.db.Candidate;
-import org.tctalent.server.model.db.CandidateCitizenship;
-import org.tctalent.server.model.db.CandidateDestination;
-import org.tctalent.server.model.db.CandidateEducation;
-import org.tctalent.server.model.db.CandidateExam;
-import org.tctalent.server.model.db.CandidateLanguage;
-import org.tctalent.server.model.db.CandidateOccupation;
-import org.tctalent.server.model.db.CandidateProperty;
-import org.tctalent.server.model.db.CandidateStatus;
-import org.tctalent.server.model.db.CandidateSubfolderType;
-import org.tctalent.server.model.db.Country;
-import org.tctalent.server.model.db.DataRow;
-import org.tctalent.server.model.db.DependantRelations;
-import org.tctalent.server.model.db.EducationLevel;
-import org.tctalent.server.model.db.Exam;
-import org.tctalent.server.model.db.Gender;
-import org.tctalent.server.model.db.HasTcQueryParameters;
-import org.tctalent.server.model.db.LanguageLevel;
-import org.tctalent.server.model.db.Occupation;
-import org.tctalent.server.model.db.PartnerImpl;
-import org.tctalent.server.model.db.QuestionTaskAssignmentImpl;
-import org.tctalent.server.model.db.Role;
-import org.tctalent.server.model.db.RootRequest;
-import org.tctalent.server.model.db.SavedList;
-import org.tctalent.server.model.db.Status;
-import org.tctalent.server.model.db.SurveyType;
-import org.tctalent.server.model.db.TaskAssignmentImpl;
-import org.tctalent.server.model.db.UnhcrStatus;
-import org.tctalent.server.model.db.UploadTaskImpl;
-import org.tctalent.server.model.db.User;
-import org.tctalent.server.model.db.YesNoUnsure;
+import org.tctalent.server.model.db.*;
```

Comment:
It looks like your Intellij is not configured for our standard code style - which follows Google's standard Java style.

Here is a screen shot of my Intellij Java CodeStyle config.
I have just highlighted the Imports tab - but the whole thing is driven by selecting Google Style in the Scheme drop down.

Then we shouldn't get all these import changes.

<img width="1402" alt="Screen Shot 2025-01-22 at 4 15 26 pm" src="https://github.com/user-attachments/assets/9f2afd17-a7b9-4ee6-b1f3-063faa68f413" />
 config.